src/dotnet/Users.Service/RecentEntriesBackend.cs

```diff
@@ -21,19 +21,19 @@ internal class RecentEntriesBackend : DbServiceBase<UsersDbContext>, IRecentEntr
         await PseudoList(shardKey, scope).ConfigureAwait(false);
 
         var list = await dbContext.RecentEntries
-            .Where(x => string.Equals(x.ShardKey, shardKey) && x.Scope == scope.ToString())
+            .Where(x => string.Equals(x.ShardKey, shardKey) && x.Scope.Equals(scope.ToString()))
```

Comment:
You can use `Equals(x.ShardKey, shardKey)`, etc. in LINQ-to-queryable to suppress the warning, + Rider actually doesn't show it for `==` there.

Reply:
![image](https://user-images.githubusercontent.com/4425748/187075256-3f75576c-5cc2-4c78-b2ce-0c452a4c37ae.png)
it shows no warnings...